tests/Feature/User/RegisterNewMemberTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Tests\Feature;
+
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Artisan;
+use TestCase;
+
+class RegisterNewMemberTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /**
+     * @var string
+     */
+    private $url = '/lidworden';
+
+    /**
+     * @var
+     */
+    private $user;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        session()->start();
+    }
+
+    protected function tearDown()
+    {
+        Artisan::call('migrate:reset');
+        parent::tearDown();
+    }
+
+    /** @test */
+    public function a_guest_can_register_a_new_user(){
+        $body = [
+            'email' => 'test@esac.nl',
+            'firstname' => 'esac',
+            'lastname' => 'esac last name',
+            'street' => 'stastion street',
+            'houseNumber' => 3,
+            'city' => 'eindhoven',
+            'zipcode' => '5781JH',
+            'country' => 'NL',
+            'phonenumber' => 28743939,
+            'emergencyNumber' => 112,
+            'emergencyHouseNumber' => '2a',
+            'emergencystreet' => 'nursery',
+            'emergencycity' => 'dead',
+            'emergencyzipcode' => '3473JP',
+            'emergencycountry' => 'NL',
+            'birthDay' => Carbon::now()->subCenturies(2)->format('d-m-Y'),
+            'gender' => 'Man',
+            'IBAN' => "BLNDj838474784848",
+            'incasso' => 1,
+            'privacy_policy' => 1,
+            'termsconditions' => 1,
+            'g-recaptcha-response' => 'ikbenvalid',
+            '_token' => csrf_token()
```

Comment:
Waarom geen faker?

Reply:
Ik gebruik faker alleen in de factories en in de tests wil ik de data wel specificeren zodat ik precies weet wat er in gaat. Want mocht het bij faker data mis gaan kan het zijn dat die de volgende keer wel slaagt